plugins/bookmarks.js

```diff
@@ -1,10 +1,18 @@
 // @author         ZasoGD
 // @name           Bookmarks for maps and portals
 // @category       Controls
-// @version        0.4.2
+// @version        0.5.0
 // @description    Save your favorite Maps and Portals and move the intel map with a click. Works with sync. Supports Multi-Project-Extension
 
 /* global L -- eslint */
+/* exported setup, changelog --eslint */
+
+var changelog = [
+  {
+    version: '0.5.0',
+    changes: ['Range display is now consistent with autodraw. Better precision for long links'],
```

Comment:
Here we only extracted functionality for use elsewhere, but the plugin functionality didn't change.

Maybe we should only bump minor version and write smth like 
```suggestion
    changes: ['(dev) extracted "formatDistance" function for global use'],
```